lib/api_browser/app/views/types/label/link.html

```diff
@@ -0,0 +1 @@
+<span>Link&nbsp;[&nbsp;<a ui-sref="root.type({version: apiVersion, type: type.link_to})">{{type.link_to | resourceName}}</a>&nbsp;]</span>
```

Comment:
If the `&nbsp;`s are to avoid whitespace being collapsed, it might be worth using the `conservativeCollapse` option for [html-minifier](https://github.com/kangax/html-minifier).